packages/manager/modules/veeam-cloud-connect/src/storage/veeam-storage.controller.js

```diff
@@ -37,54 +32,8 @@ export default class VeeamCloudConnectStorageCtrl {
     return this.CucRegionService.getTranslatedMicroRegion(region.toUpperCase());
   }
 
-  addStorage() {
-    if (this.actions.data.addStorage.available) {
-      this.CucControllerHelper.modal.showModal({
-        modalConfig: {
-          template: storageAddTemplate,
-          controller: storageAddCtrl,
-          controllerAs: 'VeeamCloudConnectStorageAddCtrl',
-          resolve: {
-            serviceName: () => this.$stateParams.serviceName,
-          },
-        },
-      })
-        .then((result) => {
-          this.VeeamCloudConnectService.startPolling(this.$stateParams.serviceName, result.data)
-            .then(this.storageInfos.load.bind(this));
-        })
-        .catch(err => this.VeeamCloudConnectService.unitOfWork.messages.push({
-          text: err.message,
-          type: 'error',
-        }));
-    } else {
-      this.CucControllerHelper.modal.showWarningModal({
-        title: this.$translate.instant('veeam_common_action_unavailable'),
-        message: this.actions.data.addStorage.reason,
-      });
-    }
-  }
-
   updateQuota(inventoryName) {
-    this.CucControllerHelper.modal.showModal({
-      modalConfig: {
-        template: updateQuotaTemplate,
-        controller: updateQuotaCtrl,
-        controllerAs: 'VeeamCloudConnectStorageUpdateQuotaCtrl',
-        resolve: {
-          inventoryName: () => inventoryName,
-          serviceName: () => this.$stateParams.serviceName,
-        },
-      },
-    })
-      .then((result) => {
-        this.VeeamCloudConnectService.startPolling(this.$stateParams.serviceName, result.data)
-          .then(this.storageInfos.load.bind(this));
-      })
-      .catch(error => this.VeeamCloudConnectService.unitOfWork.messages.push({
-        text: error.message,
-        type: 'error',
-      }));
+    this.goToStorageQuota(inventoryName);
```

Comment:
Maybe you could directly use `goToStorageQuota` ? As this function only has this purpose